waspc/data/Generator/templates/Dockerfile

```diff
@@ -25,16 +25,15 @@ RUN apk add --no-cache openssl1.1-compat
 FROM base AS server-builder
 RUN apk add --no-cache build-base libtool autoconf automake
 WORKDIR /app
+COPY server/ ./server/
 # Install npm packages, resulting in node_modules/.
-{=# usingServerPatches =}
-COPY server/patches ./server/patches
-{=/ usingServerPatches =}
-COPY server/package*.json ./server/
```

Comment:
We no longer have to selectively, or conditionally in the case of patches, copy a subset of the `server` dir over. Just copy it all, since we need the source later anyways for TS compilation.